packit_service/worker/handlers/abstract.py

```diff
@@ -398,3 +448,20 @@ def get_signature(cls, event: Event, job: Optional[JobConfig]) -> Signature:
 
     def run(self) -> TaskResults:
         raise NotImplementedError("This should have been implemented.")
+
+
+class RetriableJobHandler(JobHandler):
+    def __init__(
+        self,
+        package_config: PackageConfig,
+        job_config: JobConfig,
+        event: dict,
+        celery_task: Optional[CeleryTask] = None,
```

Comment:
`None` needs to be allowed, since we initialize the handlers already in the processing task to call some pre-check methods and there we don't want to pass the processing celery task since we retry only the actual handler task.

We discussed this with @FrNecas in chat, to resolve the None-correctness, a bigger refactoring needs to be done and Franta will create an issue with the requirements for that.